### ISSUE TITLE
Optional fields must be marked as processed.

### ISSUE THREAD

**Issue by Amartel1986** [everit-org/json-schema#250]
```
        final JSONObject rawSchema = new JSONObject("{\"title\": \"Title\"}");
        final SchemaLoader loader = SchemaLoader.builder()
            .schemaJson(rawSchema)
            .draftV7Support()
            .build();
        final Schema schema = loader.load().build();
        System.out.println(schema);
```

Results in `org.json.JSONException: Duplicate key "title"` because optional field `title` is present in `unprocessedProperties` map.

**Comment by erosb** [comment]
Thank you for reporting it, this is now fixed in `master` . This is a quite severe bug so the fix will be released fairly soon. We will see next week if I can stick any further bugfixes into the next release or will this be released on its own.

**Comment by afranzi** [comment]
Hi @erosb, do you have an estimated deadline for this new release? :)

Thanks,
Regards,

**Comment by erosb** [comment]
Hello @afranzi , you can start using the current HEAD by setting its commit hash ( `55800a1b84f1cdbbd1254cd34cd2e159167bcd81` ) as the `<version>` number (jitpack will understand it). Regarding the new release - maybe this weekend, but I can't promise anything.

**Comment by erosb** [comment]
Hello, the bugfix is included in version `1.11.0`.
